panorama/video.py

```diff
@@ -130,26 +141,10 @@ def show(self, frames: np.ndarray) -> None:
             if cv2.waitKey(1000 // self.fps) & 0xFF == ord('q'):
                 break
 
-    def overlay_image_alpha(self, img: np.ndarray, overlay: np.ndarray, x: int,
-                            y: int, alpha_mask: np.ndarray) -> np.ndarray:
+    def overlay_image_alpha(self, img: np.ndarray,
+                            overlay: np.ndarray) -> np.ndarray:
         # Image ranges
-        img = img.copy()
-        y1, y2 = max(0, y), min(img.shape[0], y + overlay.shape[0])
-        x1, x2 = max(0, x), min(img.shape[1], x + overlay.shape[1])
-
-        # Overlay ranges
-        # y1o, y2o = max(0, -y), min(overlay.shape[0], img.shape[0] - y)
-        # x1o, x2o = max(0, -x), min(overlay.shape[1], img.shape[1] - x)
-
-        # Exit if nothing to do
-        if y1 >= y2 or x1 >= x2:
-            return img
-
-        # Blend overlay within the determined ranges
-        img_crop = img[y1:y2, x1:x2]
-        mask = alpha_mask[:, :, np.newaxis]
-        # img_overlay_crop = overlay[y1o:y2o, x1o:x2o]
-        mask_inv = 1.0 - mask
-
-        img_crop[:] = mask * overlay + mask_inv * img_crop
-        return img
\ No newline at end of file
+        mask = cv2.inRange(overlay, np.array([0, 0, 0]), np.array([20, 20,
```

Comment:
why do you set 3d array with length 20?

Reply:
Actually, I can set it lower.
If upperbound is [0,0,0], foreground would have a obvious border

Reply:
gotcha, 20 should be for human right?